src/v2/Apps/Partner/Components/Overview/NearbyGalleryCard.tsx

```diff
@@ -0,0 +1,142 @@
+import { ContextModule } from "@artsy/cohesion"
+import { Box, BoxProps, ResponsiveBox, Image, Flex, Text } from "@artsy/palette"
+import { capitalize, compact, uniq } from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/Artsy"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { FollowProfileButtonFragmentContainer } from "v2/Components/FollowButton/FollowProfileButton"
+import { NearbyGalleryCard_partner } from "v2/__generated__/NearbyGalleryCard_partner.graphql"
+
+interface NearbyGalleryCardProps extends BoxProps {
+  partner: NearbyGalleryCard_partner
+  city: string | null
+}
+
+function normalizeCityName(city: string) {
+  return capitalize(city.trim())
+}
+
+function getLocation(cities: Array<string>, preferredCity: string | null) {
```

Comment:
Would you mind adding some tests for this?

Reply:
Sure, I will add it.

Reply:
Done